RandomData.Api.Tests/NumberGenerators/NumberGeneratorTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using RandomData.Api.NumberGenerators.Dto;
+using RandomData.Api.NumberGenerators.Generators;
+using Xunit;
+
+namespace RandomData.Api.Tests.NumberGenerators
+{
+    public class NumberGeneratorTests
```

Comment:
I'm afraid that in an event where something breaks we might not see the problem right away. The tests are not repeatable because they are using randomness, so for example if max value cap does not work, the test can still pass due to randomness.

Reply:
we might consider repeating generating numbers several times in a loop (f.e. 100 times) to minimize randomness in a given test

Reply:
At first I had simmilar idea in my endpoint but it's not a way to go. You should create edge points that you suspect may break something and have something like mine FakeRandom to explicitly test these edge cases. I don't think you should ever use random methods in tests because they should be repetitive.

Reply:
I don't think it's necessary to introduce fake random, probability of receiving 100 times correct answer by a coincidence is so low you can ignore it

Reply:
and it's kinda bad to modify implementation just for testing purposes

Reply:
I think that black-box techniques should only be used to as an addition not as  an only tests of an component. 

I think it @krzysztof-gwozdz might give his thought